### PR TITLE
Add 'Federal Agency' to list of accepted states

### PR DESCRIPTION
**Pull request by michplunkett**
## Fixes issue
https://github.com/lucyparsons/OpenOversight/issues/797

## Description of Changes
Added 'Federal Agency' to list of accepted states, corrected formatting of numbers on browse screen, moved choice lists to `utils` folder, and modified formatter definitions.

## Screenshots (if appropriate)
Browse:
Screenshot 2023-08-01 at 4 07 34 PM

Update:
Screenshot 2023-08-01 at 4 09 19 PM

View officer:
Screenshot 2023-08-01 at 4 09 33 PM

Header for listing officers:
Screenshot 2023-08-01 at 4 09 54 PM

## Tests and linting
 - [x] This branch is up-to-date with the `develop` branch.
 - [x] `pytest` passes on my local development environment.
 - [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

**Review by null-space-0 — COMMENTED** [review]
Overall this looks great to me! I think the example image you gave with CPD listed as a Federal Agency was just a proof of concept for the feature, correct? I can submit approval once I receive confirmation on that!

**Comment by michplunkett** [comment]
It was me just testing out values. It's updated!

**Review by null-space-0 — APPROVED** [review]
Perfect, thanks for the update :)

File: OpenOversight/tests/conftest.py [hunk 1]
```diff
@@ -38,19 +39,34 @@
...
+            else "".join(random.choices(string.ascii_uppercase + string.digits, k=20))
+        )
+
+
 OFFICERS = [
     ("IVANA", "", "TINKLE"),
     ("SEYMOUR", "", "BUTZ"),
     ("HAYWOOD", "U", "CUDDLEME"),
-    ("BEA", "", "O'PROBLEM"),
+    ("BEA", "", "PROBLEM"),
```
Comment (abandoned-prototype) [inline OpenOversight/tests/conftest.py:69]: @michplunkett was there a particular reason to change this? I believe the `O'` is in the name specifically to make sure we can handle those kind of names. Did it break a test?

Comment (michplunkett) [reply OpenOversight/tests/conftest.py:69]: None of the other last names felt like they were of one particular ethnic group and `O'PROBLEM` was one that stuck out. I'd be happy to change it to something else, but it just felt a bit off.

Comment (michplunkett) [reply OpenOversight/tests/conftest.py:69]: It's Simpsons canon apparently, I'll revert the change on the next PR!

**Pull request merged**